src/main/app/src/app/mail/ontvangstbevestiging/ontvangstbevestiging.component.ts

```diff
@@ -87,6 +100,25 @@ export class OntvangstbevestigingComponent implements OnInit {
         .documenten(documenten)
         .build();
 
+        if (initiator) {
+            this.loadingInitator = true;
+            const loadingIcon = new ActionIcon('hourglass_empty', 'msg.loading', new Subject<void>());
+            ontvanger.icons ? ontvanger.icons.push(loadingIcon) : ontvanger.icons = [loadingIcon];
+            initiator.subscribe(klant => {
```

Comment:
Het is niet de bedoeling dat de hele klant wordt opgehaald alleen maar om her email adres op te halen want:
1) Dat duurt lang vanwege de (onnodige call) naar Haal Centraal
2) Volgens de AVG is niet uit te leggen waarom je hier de gegevens van een persoon ophaalt.

Maak een methode in de KlantenService genaamd. ophalenContactgegevens welke op basis van IdentificatieType en initiatorIdentificatie de contactgegevens ophaalt bestaand uit emailadres en telefoonnummer. 

In de backend maak je in de KlantenRESTSErvice een gelijknamige methode welke alleen de KlantenClientService aanspreekt. Gebruik de findPersoon en findVestiging method

Haal dat hele lazy loading er maar af. Deze informatie komt uit Open Klant is is net zo snel beschikbaar als info uit Open Zaak

Reply:
Aangepast